Source/FieldSolver/SpectralSolver/SpectralFieldData.H

```diff
@@ -127,7 +127,7 @@ class SpectralFieldData
                                const int i_comp);
 
         void BackwardTransform (const int lev, amrex::MultiFab& mf, const int field_index,
-                                const int i_comp, const amrex::IntVect& fill_guards);
+                                const amrex::IntVect& fill_guards, const int i_comp);
```

Comment:
Here I swapped the arguments `i_comp` and `fill_guards` to match the same interface of the method `BackwardTransform` in the class `SpectralSolver`.